CODEOWNERS

```diff
@@ -54,10 +63,14 @@ src/drivers/mediatek/mt8195/*		@yaochunhung @kuanhsuncheng
 
 # other libs
 src/math/*				@singalsu
-src/ipc/*				@bardliao @marcinszkudlinski
+src/ipc/*				@bardliao @marcinszkudlinski @pblaszko
 src/lib/*				@libinyang
+src/lib-zephyr/cpu.c			@tmleman
```

Comment:
This file just changed location https://github.com/thesofproject/sof/commit/1e21a5d2fcde0fe843c63c3a43bb7d964f6d63e9

Reply:
@tmleman thanks for info, I have updated the path